src/pudl/analysis/mcoe.py

```diff
@@ -41,80 +41,98 @@
 
 def mcoe_asset_factory(
     freq: Literal["AS", "MS"],
-    io_manager_key: str | None = None,
 ) -> list[AssetsDefinition]:
     """Build MCOE related assets at yearly and monthly frequencies."""
     agg_freqs = {"AS": "yearly", "MS": "monthly"}
     if freq not in agg_freqs:
         raise ValueError(f"freq must be one of {agg_freqs.keys()}, got: {freq}.")
 
     @asset(
-        name=f"heat_rate_by_unit_{agg_freqs[freq]}",
+        name=f"_out_eia__{agg_freqs[freq]}_heat_rate_by_unit",
         ins={
             "gen": AssetIn(
                 key=f"out_eia923__{agg_freqs[freq]}_generation_fuel_by_generator_energy_source"
             ),
             "bga": AssetIn(key="core_eia860__assn_boiler_generator"),
         },
-        io_manager_key=io_manager_key,
         compute_kind="Python",
+        io_manager_key="pudl_sqlite_io_manager",
+        description=f"{agg_freqs[freq].title()} heat rate estimates by generation unit. Generation "
```

Comment:
I added the descriptions to the assets because their metadata will be removed from `pudl.metadata.resources` when we deprecate `PudlTabl`.

Reply:
This is true for other intermediate assets that are currently written to the database because we provide access to them via `PudlTabl`. These assets will use the default IO manager when we deprecate `PudlTabl`.

There are a handful of other output intermediate assets that are being written to the database that don't have descriptions specified in the asset decorators. I can remove these descriptions for now and handle it when we deprecate `PudlTabl`.